libflux/core/src/parser/tests.rs

```diff
@@ -3888,6 +3888,203 @@ fn declare_variable_as_an_empty_array() {
     )
 }
 
+#[test]
+fn parse_empty_dict() {
+    let mut p = Parser::new(r#"[:]"#);
```

Comment:
do we have an existing test that parses an empty array literal?

Reply:
Yes we do.